templates/frontend/mission/mission.html.twig

```diff
@@ -39,35 +39,60 @@
             {% endif %}
         </div>
     </div>
-    {% if can('view_after_action_reports', mission.operation) and not mission.afterActionReports.empty %}
-        <div class="mb-8">
-            <h3>{{ 'perscom.mission.aars'|trans }}</h3>
-            {{ component('Perscom\\AfterActionReportList', { mission: mission }) }}
+    <div class="flex gap-2" {{ stimulus_controller('forumify/forumify-platform/tabs') }}>
+        <div class="w-20">
```

Comment:
This does not seem mobile responsive? You should use `grid`. See `templates/frontend/components/course_class/class.html.twig` as example.